src/figuresClasses.ts

```diff
@@ -1,11 +1,91 @@
-export interface Figure {}
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
```

Comment:
Enum constant names should start with a capital letter. Please adjust 'Triangle', 'Circle', and 'Rectangle' to start with capital letters.